src/web-components.js

```diff
@@ -48,6 +48,28 @@ export function isShadowDomSupported() {
 }
 
 /**
+ * Returns `true` if Shadow CSS encapsulation is supported.
+ * @return {boolean}
+ */
+export function isShadowCssSupported() {
+  if (!isShadowDomSupported()) {
+    return false;
+  }
+
+  //TODO: Find a better way to test CSS encapsulation
```

Comment:
Add `DO NOT SUBMIT` here.

I think our best bet for now is to check if `toString(shadowRoot) == '[object ShadowRoot]'`. We need to check if this is the case for Shady and WebComponents polyfills.